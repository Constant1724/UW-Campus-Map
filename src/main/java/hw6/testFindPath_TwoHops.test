# START_TEST
# NAME TwoHops
# test a graph can be loaded and correctly find the path from A to B.

# Create a graph
CreateGraph graph1

# AddNode
AddNode graph1 ValidNode1
AddNode graph1 ValidNode2
AddNode graph1 ValidNode3
AddNode graph1 ValidNode4
AddNode graph1 ValidNode5
AddNode graph1 ValidNode6
AddEdge graph1 ValidNode1 ValidNode2 label12
AddEdge graph1 ValidNode1 ValidNode3 label13
AddEdge graph1 ValidNode1 ValidNode4 label14
AddEdge graph1 ValidNode3 ValidNode5 label35
AddEdge graph1 ValidNode5 ValidNode6 label56

FindPath graph1 ValidNode1 ValidNode5

# END_TEST