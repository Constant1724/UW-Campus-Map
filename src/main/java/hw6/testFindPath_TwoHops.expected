# START_TEST
# NAME TwoHops
# test a graph can be loaded and correctly find the path from A to B.

# Create a graph
created graph graph1

# AddNode
added node ValidNode1 to graph1
added node ValidNode2 to graph1
added node ValidNode3 to graph1
added node ValidNode4 to graph1
added node ValidNode5 to graph1
added node ValidNode6 to graph1
added edge label12 from ValidNode1 to ValidNode2 in graph1
added edge label13 from ValidNode1 to ValidNode3 in graph1
added edge label14 from ValidNode1 to ValidNode4 in graph1
added edge label35 from ValidNode3 to ValidNode5 in graph1
added edge label56 from ValidNode5 to ValidNode6 in graph1

path from ValidNode1 to ValidNode5:
ValidNode1 to ValidNode3 via label13
ValidNode3 to ValidNode5 via label35

# END_TEST